colossalai/nn/parallel.py

```diff
@@ -3,8 +3,10 @@
 from colossalai.core import global_context as gpc
 from colossalai.context import ParallelMode
 from functools import partial
+from colossalai.zero.utils.zero_hook_v2 import ZeROHookV2
+from colossalai.tensor import ChunkManager, use_param_op_hooks, TensorState
 
-__all__ = ['ColoDDP']
+__all__ = ['ColoDDP', 'ColoDDPV2']
```

Comment:
I'm not familiar with previous zero. What DDPV2 means?

Reply:
`ColoDDP` is like torch ddp, which only supports traditional DP. `ColoDDPV2` supports ZeRO-DP and traditional DP, so I still name it `DDP`. `ColoDDP` and `ColoDDPV2` are both for `ColoTensor`, so I name the second one `V2`.